onyx-database-tests/pom.xml

```diff
@@ -5,7 +5,7 @@
 
 
     <modelVersion>4.0.0</modelVersion>
-    <name>onyx-database-tests</name>
+    <name>${project.groupId}:${project.artifactId}</name>
```

Comment:
This is how the Maven Central guide recommends doing it.  See: http://central.sonatype.org/pages/requirements.html